pySmartDL/pySmartDL.py

```diff
@@ -75,14 +77,21 @@ class SmartDL:
             * If no path is provided, `%TEMP%/pySmartDL/` will be used.
     '''
     
-    def __init__(self, urls, dest=None, progress_bar=True, fix_urls=True, threads=5, timeout=5, logger=None, connect_default_logger=False):
+    def __init__(self, urls, dest=None, progress_bar=True, fix_urls=True, threads=5, timeout=5, logger=None, connect_default_logger=False, requestArgs=None):
```

Comment:
1. this is user-facing API and therefore requestArgs should become request_args
2. I feel like it should come after "fix_urls" or something, and not at the end of the constructor

Reply:
1 and 3 have been fixed. As for 2, I put it at the end as to not break any code that currrently uses positional arguments. I can change it if you'd like though.